media/js/src/components/extendeddatevue.js

```diff
@@ -118,35 +115,14 @@ var ExtendedDateVue = {
         });
         $el.find('[data-toggle="tooltip"]').tooltip();
 
-        if (this.approximate1 === 'True') {
-            $el.find('input.approximate').prop('checked', true);
-        }
-        if (this.uncertain1 === 'True') {
-            $el.find('input.uncertain').prop('checked', true);
-        }
-        if (this.millenium1) {
-            $el.find('.millenium').val(this.millenium1);
-        }
-        if (this.century1) {
-            $el.find('.century').val(this.century1);
-        }
-        if (this.decade1) {
-            $el.find('.decade').val(this.decade1);
-        }
-        if (this.year1) {
-            $el.find('.year').val(this.year1);
-        }
-        if (this.month1) {
-            const selector = '.month option[value=' +
-                this.month1.replace(/^[0]+/g, '') + ']';
-            $el.find(selector).attr('selected','selected');
-        }
-        if (this.day1) {
-            $el.find('.day').val(this.day1);
-        }
-
-        if (this.initialErrors) {
-            this.errors = 1;
-        }
+        this.approximate1 = this.initialApproximate1;
```

Comment:
Interesting note that Vue converts kebab-case to camel-case automagically. So, the prop `initial-uncertain1` becomes `this.initialUncertain1`. Kind of mysterious and non-obvious, but nice all the same.